app/authentication/jti_claim_storage.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, jti_claim):
         self.jti_claim = jti_claim
 
     def __str__(self, *args, **kwargs):
-        return "jti claim '{jti_claim}' has already been used".format(jti_claim=self.jti_claim)
+        return 'jti claim \'{jti_claim}\' has already been used'.format(jti_claim=self.jti_claim)
```

Comment:
Google's Python style guide suggests permitting using the alternate quote style if the string contains a quote i.e. use `"` if `'` is used in the string to avoid the backslashes - https://google.github.io/styleguide/pyguide.html?showone=Strings#Strings

Reply:
Looks like the package we use to enforce these checks can be configured for "smart" checking with `single-avoid-escape`. I'll update the config to use that.